.golangci.yaml

```diff
@@ -144,6 +144,8 @@ linters-settings:
         alias: ${1}helper
       - pkg: github.com/gardener/gardener/pkg/apis/([^c]\w+)/([\w\d]+)
         alias: $1$2
+      - pkg: github.com/gardener/gardener/pkg/apis/([^c]\w+)/([\w\d]+)/([\w\d]+)
+        alias: $1$2$3
```

Comment:
WDYT about

```yaml
      - pkg: github.com/gardener/gardener/pkg/apis/resources/([\w\d]+)/helper
        alias: resources$1helper
```

?

I was just wondering whether the current revision is too much opinionated.

Reply:
This way it's applicable for seedmanagement as well. Did you find any other import which wouldn't feel right with this alias?

Reply:
Anyways